modules/instance_template/metadata.yaml

```diff
@@ -223,6 +223,27 @@ spec:
         description: Metadata, provided as a map
         varType: map(string)
         defaultValue: {}
+        connections:
+          - source:
+              source: github.com/terraform-google-modules/terraform-google-memorystore
+              version: "~> 10.0.0"
```

Comment:
Could we pin to major versions like `"~> 10.0` so it connects with any 10.x release (here and below). Any change ti outputs would be a breaking change and should result in a new major version.

Reply:
Thanks! Done!